mappings/net/minecraft/client/options/GameOptions.mapping

```diff
@@ -95,6 +95,8 @@ CLASS net/minecraft/class_315 net/minecraft/client/options/GameOptions
 	FIELD field_1915 chatWidth D
 	FIELD field_1916 soundVolumeLevels Ljava/util/Map;
 	FIELD field_19244 discreteMouseScroll Z
+	FIELD field_21332 sneakToggled Z
+	FIELD field_21333 sprintToggled Z
```

Comment:
What does "toggled" mean here?

Reply:
so now sneak and sprint can be sticky keys. when they are toggled they are considered always pressed. See accessibility options.